tools/tpm2_nvdefine.c

```diff
@@ -195,14 +195,13 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
         { "index",                  required_argument,  NULL,   'x' },
         { "hierarchy",              required_argument,  NULL,   'a' },
         { "size",                   required_argument,  NULL,   's' },
-        { "attributes",             required_argument,  NULL,   't' },
+        { "index-attributes",       required_argument,  NULL,   'b' },
```

Comment:
I think we just want attributes. I think <specifier>-attributes is to much, and I plan on doing a round of long-option name sanity after we get the short options squared away. I wouldn't bother changing this at the moment.